src/diffusers/schedulers/scheduling_dpmsolver_multistep.py

```diff
@@ -140,6 +147,8 @@ def __init__(
         solver_type: str = "midpoint",
         lower_order_final: bool = True,
         use_karras_sigmas: Optional[bool] = False,
+        lambda_min_clipped: float = -5.1,
```

Comment:
For backwards compatibility, could we set this to `-float("inf")` so that we can be sure that by default nothing is changed? 

```suggestion
        lambda_min_clipped: float = -inf,
```